### PR TITLE
Add release 1.12.3 (blog post)

### PR DESCRIPTION
**Pull request by AHeise**

### CODE REVIEW

**Review by dianfu — COMMENTED** [review]
@AHeise I guess we should also update the following pages:
- _config.yaml
- q/gradle-quickstart.sh, q/quickstart-scala.sh, q/quickstart.sh, q/sbt-quickstart.sh 

Could refer to https://github.com/apache/flink-web/pull/405 for more details.

**Comment by AHeise** [comment]
> @AHeise I guess we should also update the following pages:
> 
>     * _config.yaml
> 
>     * q/gradle-quickstart.sh, q/quickstart-scala.sh, q/quickstart.sh, q/sbt-quickstart.sh
> 
> 
> Could refer to #405 for more details.

Thank you. I have updated my PR.

I was not too sure if I needed to update both ` q/gradle-quickstart.sh ` and ` content/q/gradle-quickstart.sh ` etc.

File: _config.yml [hunk 1]
```diff
@@ -58,32 +58,32 @@ flink_releases:
   -
     version_short: "1.12"
     binary_release:
-      name: "Apache Flink 1.12.2"
+      name: "Apache Flink 1.12.3"
       scala_211:
         id: "1122-download_211"
```
Comment (dianfu) [inline _config.yml]:
```suggestion
        id: "1123-download_211"
```

File: _config.yml [hunk 2]
```diff
@@ -58,32 +58,32 @@ flink_releases:
...
-        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.2/flink-1.12.2-bin-scala_2.11.tgz"
-        asc_url: "https://downloads.apache.org/flink/flink-1.12.2/flink-1.12.2-bin-scala_2.11.tgz.asc"
-        sha512_url: "https://downloads.apache.org/flink/flink-1.12.2/flink-1.12.2-bin-scala_2.11.tgz.sha512"
+        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.3/flink-1.12.3-bin-scala_2.11.tgz"
+        asc_url: "https://downloads.apache.org/flink/flink-1.12.3/flink-1.12.3-bin-scala_2.11.tgz.asc"
+        sha512_url: "https://downloads.apache.org/flink/flink-1.12.3/flink-1.12.3-bin-scala_2.11.tgz.sha512"
       scala_212:
         id: "1122-download_212"
```
Comment (dianfu) [inline _config.yml]: ditto

File: _config.yml [hunk 3]
```diff
@@ -58,32 +58,32 @@ flink_releases:
...
-        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.2/flink-1.12.2-bin-scala_2.12.tgz"
-        asc_url: "https://downloads.apache.org/flink/flink-1.12.2/flink-1.12.2-bin-scala_2.12.tgz.asc"
-        sha512_url: "https://downloads.apache.org/flink/flink-1.12.2/flink-1.12.2-bin-scala_2.12.tgz.sha512"
+        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.3/flink-1.12.3-bin-scala_2.12.tgz"
+        asc_url: "https://downloads.apache.org/flink/flink-1.12.3/flink-1.12.3-bin-scala_2.12.tgz.asc"
+        sha512_url: "https://downloads.apache.org/flink/flink-1.12.3/flink-1.12.3-bin-scala_2.12.tgz.sha512"
     source_release:
-      name: "Apache Flink 1.12.2"
+      name: "Apache Flink 1.12.3"
       id: "1122-download-source"
```
Comment (dianfu) [inline _config.yml]: ditto

File: _config.yml [hunk 4]
```diff
@@ -58,32 +58,32 @@ flink_releases:
...
-      sha512_url: "https://downloads.apache.org/flink/flink-1.12.2/flink-1.12.2-src.tgz.sha512"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.3/flink-1.12.3-src.tgz"
+      asc_url: "https://downloads.apache.org/flink/flink-1.12.3/flink-1.12.3-src.tgz.asc"
+      sha512_url: "https://downloads.apache.org/flink/flink-1.12.3/flink-1.12.3-src.tgz.sha512"
     optional_components:
       -
         name: "Avro SQL Format"
         category: "SQL Formats"
         scala_dependent: false
         id: 1122-sql-format-avro
```
Comment (dianfu) [inline _config.yml]: ditto

File: _config.yml [hunk 5]
```diff
@@ -206,7 +206,7 @@ component_releases:
 release_archive:
     flink:
       - version_short: "1.12"
-        version_long: 1.12.2
+        version_long: 1.12.3
         release_date: 2021-03-03
```
Comment (dianfu) [inline _config.yml:215]: Please remember to update the date when the release passes.

**Review by dianfu — APPROVED** [review]

**Review by dawidwys — APPROVED** [review]

**Pull request merged**